### DISCUSSION TITLE
Using `black` on my `$PATH`

### DISCUSSION THREAD

**Discussion by parsiad** [LazyVim/LazyVim#2046] [Q&A] [answered]
I have enabled the `formatting.black` LazyVim Extra. When I use it, it always prefers `$HOME/.local/share/nvim/mason/bin/black` but I would like it to prefer the `black` on my path (in my case, populated from a direnv configuration) which is a different version. A simple fix that I have found works is to manually move `$HOME/.local/share/nvim/mason/bin/black`. Is there a cleaner way to achieve the same effect?

**Comment by dpetka2001** [comment, marked as answer]
You're not specifying which formatter you use, so I'm just going to assume that you're using LazyVim's default, which is `conform`.
Inside the `conform` spec you could do something like `opts.formatters.black.command = /path/to/your/local/black` to specify which `black` binary should be used.
